src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketsTelemetry.cs

```diff
@@ -113,7 +113,7 @@ public void AcceptStart(EndPoint address)
         {
             if (IsEnabled(EventLevel.Informational, EventKeywords.All))
             {
-                AcceptStart(address.ToString());
+                AcceptStart(address.Serialize().ToString());
```

Comment:
This makes it so that both Sync and Async `AcceptStart` events use the same format for the address.